tasks/winbuildscripts/sysprobe.ps1

```diff
@@ -12,8 +12,7 @@ if ($Env:TARGET_ARCH -eq "x64") {
 $PROBE_BUILD_ROOT=(Get-Location).Path
 $Env:PATH="$PROBE_BUILD_ROOT\dev\lib;$Env:GOPATH\bin;$Env:Python2_ROOT_DIR;$Env:Python2_ROOT_DIR\Scripts;$Env:Python3_ROOT_DIR;$Env:Python3_ROOT_DIR\Scripts;$Env:PATH"
 
-& $Env:Python3_ROOT_DIR\python.exe -m pip install PyYAML==5.3.1
-& pip install awscli==1.19.112 --upgrade
+& $Env:Python3_ROOT_DIR\python.exe -m pip install awscli==1.19.112 --upgrade
```

Comment:
does `pip` run a different pip than `& $Env:Python3_ROOT_DIR\python.exe -m pip` ?

the sysprobe test is failing now on missing awscli

Reply:
Mmmmmh interesting, let me try to change this !

Reply:
Interesting I didn't realise my change changed the `pip` we were using ! Changing this to see if it fixes it !